streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/ClientState.java

```diff
@@ -16,28 +16,31 @@
  */
 package org.apache.kafka.streams.processor.internals.assignment;
 
-import org.apache.kafka.common.TopicPartition;
-import org.apache.kafka.streams.processor.TaskId;
-import org.apache.kafka.streams.processor.internals.Task;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import static java.util.Collections.emptyMap;
+import static java.util.Collections.unmodifiableMap;
+import static java.util.Collections.unmodifiableSet;
+import static java.util.Comparator.comparing;
+import static org.apache.kafka.common.utils.Utils.union;
+import static org.apache.kafka.streams.processor.internals.assignment.SubscriptionInfo.UNKNOWN_OFFSET_SUM;
 
 import java.util.Collection;
 import java.util.Comparator;
+import java.util.EnumMap;
 import java.util.HashSet;
 import java.util.Map;
+import java.util.Map.Entry;
 import java.util.Set;
-import java.util.stream.Collectors;
 import java.util.TreeMap;
 import java.util.TreeSet;
 import java.util.UUID;
+import java.util.function.Function;
+import java.util.stream.Collectors;
 
-import static java.util.Collections.emptyMap;
```

Comment:
Looks like you're IDE might have a conflicting style for import order. There's no "official" import order in the project, but in general we put static imports last rather than first. 

There's actually an ongoing KIP/PR for cleaning this up but obviously it's a pretty big undertaking, so it's been in limbo a while. For now it may be best to just avoid reordering any of the imports, and just follow the existing order in the file